src/Chapter1.hs

```diff
@@ -641,7 +651,9 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit n
+  | (abs n) < 10 = abs n
+  | (abs n) >= 10 = firstDigit(div n 10)
```

Comment:
That is right 💯 

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼 

Also in Haskell, the function and arguments are always separated by space even when the argument is in parenthesis 🙂 E.g. `firstDigit (div n 10)`